beta/src/components/MDX/Sandpack/SandpackRoot.tsx

```diff
@@ -74,7 +74,7 @@ function SandpackRoot(props: SandpackProps) {
 
   files['/styles.css'] = {
     code: [sandboxStyle, files['/styles.css']?.code ?? ''].join('\n\n'),
-    hidden: !files['/styles.css']?.visible,
+    hidden: files['/styles.css']?.hidden,
```

Comment:
`files` used to be `any` since `createFileMap` returned `any`. Now that we type the input of `createFileMap`, TypeScript is able to infer a more accurate return type of `createFileMap`.